### PR TITLE
Added section for troubleshooting updates

### PR DESCRIPTION
**Pull request by harryi3t**
TODO:
- [x] Upload the screenshots to s3 and use them

### CODE REVIEW

**Comment by harryi3t** [comment]
@kaustavdm yes, I had created a TODO for the same thing.
I wanted to get the doc reviewed first and then only upload these images to the s3 bucket since I don't have access it would be roundtrips would be expensive in case there are issues with the content/images.

File: v6/postman/launching_postman/installation_and_updates.md [hunk 1]
```diff
@@ -186,6 +186,40 @@ Some users may encounter issues updating in the Windows version of Postman 6.1.
 
 [![step 3](https://assets.postman.com/postman-docs/Troubleshootwindows6.1+(1).png)](https://assets.postman.com/postman-docs/Troubleshootwindows6.1+(1).png)
 
+## Troubleshooting Postman Updates
+
+### Update failed error
+
+While updating the app there might be some issues due to which the user would see the following modal
```
Comment (kaustavdm) [inline v6/postman/launching_postman/installation_and_updates.md]: Missing punctuation at end of line.

File: v6/postman/launching_postman/installation_and_updates.md [hunk 2]
```diff
@@ -186,6 +186,40 @@ Some users may encounter issues updating in the Windows version of Postman 6.1.
...
+
+  [![multiple-updates-running][multiple-updates-running]][multiple-updates-running]
+
+[write-permission-issue]: https://user-images.githubusercontent.com/5207331/66752338-d1050c80-eeae-11e9-9568-e20b030ee41d.png
+[multiple-updates-running]: https://user-images.githubusercontent.com/5207331/66752179-779cdd80-eeae-11e9-8514-30bb1f201452.png
+[update-error-dialog]: https://user-images.githubusercontent.com/5207331/66751141-f47a8800-eeab-11e9-8034-f268fbd2c674.png
+
+### Update button not available
+
+Users who have installed Postman using [Snap](https://snapcraft.io/postman) would see that there is no `Check for updates` button. This is because the updates are handled by Snap itself and it would automatically update Postman on a regular cadence. It you are on Postman v6, you would have to migrate your team to Postman v7 and then change the Snap channel to get the latest updates. For more information see [Migrating to Postman v7](/docs/postman_pro/managing_postman_pro/migrating_to_v7).
```
Comment (kaustavdm) [inline v6/postman/launching_postman/installation_and_updates.md]: "_If_ you are on Postman v6"

**Comment by kaustavdm** [comment]
Content looks okay to me. I'll wait to approve the PR till the images are in place.

**Pull request merged**